web/client-api/src/main/java/io/deephaven/web/public/freeze.html

```diff
@@ -40,16 +39,17 @@ <h3>Selected table data</h3>
 <button id="next_frozen">&gt;</button>
 <button id="end_frozen">&gt;&gt;</button>
 
-<script>
-
+<script type="module">
+    import {dh} from './dh-core.js';
```

Comment:
Ideally we'd be able to destructure at import time, i.e. `import { Client } from './dh-core.js';`, but I think as @niloc132 explained that would be difficult/impossible to do with GWT, so this should be fine - just doing `import dh from './dh-core.js';`